macs.php

```diff
@@ -0,0 +1 @@
+i<?php include 'data.php';  ?>
```

Comment:
GitHub PRs - the ideal place to catch up with old friends :)

Not sure if I've ever removed a channel from my IRC client... And yeah, see you at the ccc!